# Type: fix
# Subtype: I3
# English Summary: Added CIDR protocol and community device detection
# Chinese Summary: 添加cidr协议和社区设备判断
# Entry Number: 3

# === Git Diff Content ===

diff --git a/platform/platform-impl/src/com/intellij/help/impl/MacHelpUtil.java b/platform/platform-impl/src/com/intellij/help/impl/MacHelpUtil.java
index 0ded3e8..88eb3b0 100644
--- a/platform/platform-impl/src/com/intellij/help/impl/MacHelpUtil.java
+++ b/platform/platform-impl/src/com/intellij/help/impl/MacHelpUtil.java
@@ -19,6 +19,7 @@ import com.intellij.openapi.util.SystemInfo;
 import com.intellij.openapi.util.registry.Registry;
 import com.intellij.ui.mac.foundation.Foundation;
 import com.intellij.ui.mac.foundation.ID;
+import com.intellij.util.PlatformUtils;
 import org.jetbrains.annotations.NonNls;
 
 /**
@@ -39,6 +40,6 @@ public class MacHelpUtil {
   }
 
   static boolean isApplicable() {
-    return SystemInfo.isMac && Registry.is("ide.mac.show.native.help", false); 
+    return SystemInfo.isMac && Registry.is("ide.mac.show.native.help", false) && !PlatformUtils.isCidr() && !PlatformUtils.isCommunity();
   }
 }
